package.json

```diff
@@ -15,14 +15,18 @@
     "@mui/x-charts": "^7.15.0",
     "@supabase/supabase-js": "^2.45.3",
     "next": "14.2.7",
+    "nodemailer": "^6.9.15",
     "react": "^18",
     "react-dom": "^18",
+    "resend": "^4.0.0",
```

Comment:
Since we are no longer using resend and instead using nodemailer let's remove this from the package.json

Reply:
Got it!